project/src/com/google/daggerqueryui/page.html

```diff
@@ -0,0 +1,39 @@
+<!--Copyright 2020 Google LLC-->
+
+<!--Licensed under the Apache License, Version 2.0 (the "License");-->
+<!--you may not use this file except in compliance with the License.-->
+<!--You may obtain a copy of the License at-->
+
+<!--https://www.apache.org/licenses/LICENSE-2.0-->
+
+<!--Unless required by applicable law or agreed to in writing, software-->
+<!--distributed under the License is distributed on an "AS IS" BASIS,-->
+<!--WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.-->
+<!--See the License for the specific language governing permissions and-->
+<!--limitations under the License.-->
+
+<html>
+<head>
+    <title>Dagger Query UI</title>
+
+    <link rel="stylesheet" href="page-style.css">
+    <link rel="stylesheet" href="third-party/css/bootstrap.min.css">
+    <link rel="stylesheet" href="third-party/css/mdb.min.css">
+    <link rel="stylesheet" href="third-party/css/roboto-fonts.css">
+
+    <!-- JQuery -->
+    <script src="third-party/js/jquery.min.js"></script>
+    <!-- MDB core JavaScript -->
+    <script type="text/javascript" src="third-party/js/mdb.min.js"></script>
```

Comment:
Super nit: One script tag has the attribute `type="text/javascript"` and the other doesn't. That said, as JavaScript is the default programming language of the browser the attribute isn't needed.

Reply:
Thanks, I have removed it. In general, [as I see](https://stackoverflow.com/questions/21098865/text-javascript-vs-application-javascript) `text/javascript` is obsolete and people recommend to use `application/javascript` instead of it - is there any cases when it makes sense or specifying `javascript` in `type` attribute is always redundant?

Reply:
Yep, if you're only targetting modern browsers then the `type` (MIME type) attribute is always redundant. The script type if inferred via [packet inspection](https://mimesniff.spec.whatwg.org/) by the browser (due to a lack of standardization). In HTML5 compliant browsers an ommited MIME type in a script tag indicates the script is JavaScript. As a result the spec urges authors to omit the attribute rather than provide a redundant MIME type.

LGTM!